src/ontology/cl-edit.owl

```diff
@@ -19412,10 +19412,11 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002603 obo:CL_0000127
 
 # Class: obo:CL_0002604 (astrocyte of the hippocampus)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-7258-9596"^^xsd:string) obo:IAO_0000115 obo:CL_0002604 "An astrocyte that is located in the hippocampus.")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0002604 <https://orcid.org/0000-0003-1980-3228>)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002604 "2011-03-06T06:57:33Z"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002604 "astrocyte of the hippocampus"^^xsd:string)
-EquivalentClasses(obo:CL_0002604 ObjectIntersectionOf(obo:CL_0000127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001954)))
+EquivalentClasses(obo:CL_0002604 ObjectIntersectionOf(obo:CL_0000127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002421)))
```

Comment:
[astrocyte of the hippocampus](http://purl.obolibrary.org/obo/CL_0002604) EquivalentTo [astrocyte](http://purl.obolibrary.org/obo/CL_0000127) and ([part_of](http://purl.obolibrary.org/obo/BFO_0000050) some [hippocampal formation](http://purl.obolibrary.org/obo/UBERON_0002421))

To consider: label is 'astrocyte of the hippocampus' and EA is astrocyte and part_of some 'hippocampal formation', so adding a synonym "astrocyte of the hippocampal formation" and/or comment as to why the label suggest something slightly different from the EA.

Reply:
I like label as hippocampal astrocyte - this will probably be more consistent with literature too, esp with single cell work that is upcoming I think